WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -3,30 +3,30 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="title" content="Wasabi Wallet - Unfairly Private">
+    <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
```

Comment:
IMO the `with built-in CoinJoin` isn't something you want to talk about. It's just an implementation detail.

Reply:
Let's have Bitcoin privacy wallet

Reply:
Oh, I see that this was used in multiple places - and consistency is more important. @RiccardoMasutti this is your call - you can leave it as it is or shorten whatever you decide it will be used.

Reply:
I think we should optimize Wasabi's website as possible.
Google shows desktop titles to 70-71 characters and mobile titles to 78 characters. It usually truncate titles that are longer than these numbers.

With this PR we have 61 characters and every needed keyword inside it, so it would be perfect.